backend/src/packages/stripe/stripe.service.ts

```diff
@@ -0,0 +1,231 @@
+      });
+    }
+  }
+
+  public async processWebhook(event: Stripe.DiscriminatedEvent): Promise<void> {
+    switch (event.type) {
+      case StripeEvent.CAPABILITY_UPDATED: {
+        {
+          if (event.account && event.data.object.status === 'active') {
+            // Linked account has been activated
+            const business = await this.businessService.findByStripeId(
+              event.account,
+            );
+
+            if (business && !business.stripeActivated) {
+              await this.businessService.update({
+                id: business.id,
+                payload: { stripeActivated: true },
+              });
+            }
+          }
+        }
+        break;
+      }
+      case StripeEvent.PAYMENT_INTENT_SUCCEEDED: {
+        // Handle successfull payment
+        // TODO: Currently there is no logic here, but we can add a webhook on successful payment later
+        return;
+      }
+      default:
+    }
+  }
+
+  public async generateCheckoutLink(
+    order: OrderResponseDto,
+  ): Promise<string | null> {
+    const business = await this.businessService.findById(order.businessId);
+
+    if (!business) {
+      throw new HttpError({
+        status: HttpCode.BAD_REQUEST,
+        message: HttpMessage.BUSINESS_DOES_NOT_EXIST,
+      });
+    }
+
+    if (!order.shift.truck) {
+      throw new HttpError({
+        status: HttpCode.BAD_REQUEST,
+        message: HttpMessage.TRUCK_DOES_NOT_EXIST,
+      });
+    }
+    const truck = await this.truckService.findById(order.shift.truck.id);
+
+    if (!truck) {
+      throw new HttpError({
+        status: HttpCode.BAD_REQUEST,
+        message: HttpMessage.TRUCK_DOES_NOT_EXIST,
+      });
+    }
+
+    if (!business.stripeActivated || !business.stripeId) {
+      throw new HttpError({
+        status: HttpCode.BAD_REQUEST,
+        message: HttpMessage.BUSINESS_STRIPE_NOT_ACTIVATED,
+      });
+    }
+
+    const metadata: CheckoutMetadata = {
+      businessId: business.id,
+      orderId: order.id,
+      customerName: order.customerName,
+      customerPhone: order.customerPhone,
+      userId: order.userId,
+    };
+
+    const distanceInMeters = await this.mapService.getDistance(
+      order.startPoint,
+      order.endPoint,
+    );
+
+    return await this.stripeRepository.createCheckoutSession({
+      businessStripeId: business.stripeId,
+      distance: convertMetersToKm(distanceInMeters.value),
+      pricePerUnit: truck.pricePerKm,
+      metadata,
+    });
+  }
+
+  public async getPayments(
+    user: UserEntityObjectWithGroupT,
+    options: GetPaymentsRequest,
+  ): Promise<GetPaymentsResponse> {
+    const userBusinessId = await this.checkUserBusinessAndGetId(user);
+
+    if (userBusinessId) {
+      options.businessId = userBusinessId;
+    } else {
+      options.userId = user.id;
+    }
+
+    const items = await this.stripeRepository.collectPaymentIntents(options);
+
+    const page = options.page ?? 1;
```

Comment:
Тhis is not a magic number. Тhis is a page one. It can't be ambiguous here, I can even show it to my 5y.o. son and he will exlain it's meaning.

Naming it ONE or FIRST_PAGE will not make it clearer ever so slightly.

P.S. I know there might be some cases when it is not quite self-explanatory, but honestly, this is not one one of them, cmon :)